src/App.tsx

```diff
@@ -17,7 +42,9 @@ const App = () => {
     <div className={styles.mainContainer}>
       <h1 className={styles.header}>Welcome to Miracle Flights!</h1>
       <button onClick={handleAddFlight}>Add new flight!</button>
-      {/* Put the table here and pass in the flights */}
+      {
```

Comment:
No need for the `{` here and the `}` on line 47! We can treat our components just like regular html tags